web/src/main/webapp/xslt/services/sitemap/sitemap.xsl

```diff
@@ -81,7 +81,7 @@
               <xsl:value-of select="concat($nodeUrl, 'api/sitemap?', $formatParam, 'doc=', $pStart)"/>
             </loc>
             <lastmod>
-              <xsl:value-of select="$changeDate"/>
+              <xsl:value-of select="substring($changedate,1,10)"/>
```

Comment:
This should be `$changeDate` instead of `$changedate`, otherwise fails with:

Error at xsl:value-of on line 84 column 67 of sitemap.xsl:
  XPST0008: XPath syntax error at char 21 on line 84 in {substring($changedate,1}:
    Variable $changedate has not been declared